apps/server/src/modules/h5p-editor/h5p-editor-test.module.ts

```diff
@@ -11,6 +11,7 @@ import { LoggerModule } from '@src/core/logger';
 import { AuthenticationApiModule } from '@src/modules/authentication/authentication-api.module';
 import { AuthenticationModule } from '@src/modules/authentication/authentication.module';
 import { AuthorizationModule } from '@src/modules/authorization';
+import { UserRepo } from '@shared/repo';
```

Comment:
Please do not use repo directly. You need to import userModule and use user.service

Reply:
@SevenWaysDP Changes were implemented.